tests/phpunit/tests/option/option.php

```diff
@@ -569,20 +705,22 @@ static function () use ( $default_value ) {
 			}
 		);
 
-		update_option( $option, $default_value );
+		$this->assertTrue( update_option( $option, $default_value ), 'update_option() should have returned true.' );
```

Comment:
In the old `update_option()`, if the old value and new value were strictly equal, it would return `false`, whether or not the old value matched the default. [See here](https://github.com/WordPress/wordpress-develop/blob/6.2/src/wp-includes/option.php#L501).

In this test:
- The option doesn't exist yet.
- The default value has been filtered to `default-value`.
- The new value is `default-value`.

So this test will actually return `false`.

Since this is supposed to be testing that a non-existent option with a filtered default value is added to the database, we need to make a couple of changes so that the new value isn't the same as the default value.

Change 1:
```suggestion
		$this->assertTrue( update_option( $option, 'new value' ), 'update_option() should have returned true.' );
```

Reply:
Unresolving so it's available for reference.